src/funcCoreTools/getBrewPackageName.ts

```diff
@@ -5,7 +5,37 @@
 
 import { funcPackageName } from '../constants';
 import { FuncVersion, getMajorVersion } from '../FuncVersion';
+import { cpUtils } from '../utils/cpUtils';
 
 export function getBrewPackageName(version: FuncVersion): string {
     return `${funcPackageName}@${getMajorVersion(version)}`;
 }
+
+export async function tryGetInstalledBrewPackageName(version: FuncVersion): Promise<string | undefined> {
```

Comment:
While `isBrewPackageInstalled` is generic-purpose, `tryGetInstalledBrewPackageName` is only specifically for Function Core Tools, right?  The naming is a little confusing to me.

Reply:
Huh yeah I guess `isBrewPackageInstalled` is generic, but it's not exported from this file and it's only really used for Function Core Tools